### PR TITLE
Release 5.2.2

### PR DESCRIPTION
**Pull request by avokhmin**
Next release of `friendly_id`

### CODE REVIEW

**Comment by parndt** [comment]
It's confusing that this fails CI

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ gemfile:
   - gemfiles/Gemfile.rails-4.2.rb
   - gemfiles/Gemfile.rails-5.0.rb
 
+before_install:
+  - gem update bundler
```
Comment (avokhmin) [inline .travis.yml]: > It's confusing that this fails CI
 
@parndt , the problem was here. Travis used very old version of `bundler ` gem.

See https://travis-ci.org/norman/friendly_id/jobs/275578558#L982 :

```
$ bundle --version
Bundler version 1.8.4
```

**Comment by parndt** [comment]
Released to rubygems.org, thanks @avokhmin !

**Comment by avokhmin** [comment]
@parndt , thanks 🍻

**Pull request merged**